packages/xod-cli/README.md

```diff
@@ -9,23 +9,245 @@ other packages. The responsibility of `xod-cli` is to parse command line
+  $ xodc publish
+  Publish a project saved as xodball
+  $ xodc publish foo.xodball
+```
+
+_See code: [src/commands/publish.js](https://github.com/xodio/xod/blob/master/packages/xod-cli/src/commands/publish.js)_
+
+## `xodc resave [options] [project]`
+
+opens a project and saves it in another location or format
+
+```
+USAGE
+  $ xodc resave [options] [project]
+
+ARGUMENTS
+  PROJECT
+      Project to operate on. The project should point to a file or directory
+      on file system. If omitted, it is inferred from the current working
+      directory. Examples:
+
+         * ./path/to/proj.xodball           # xodball
+         * ./path/to/proj                   # just full path to a project
+
+OPTIONS
+  -V, --version              show CLI version
+  -h, --help                 show CLI help
+  -o, --output=output        xodball or multifile directory output path, defaults to stdout
+  -q, --quiet                do not log messages other than errors
+  -w, --workspace=workspace  [default: /home/sk/xod] use the workspace specified, defaults to $HOME/xod
+
+EXAMPLES
+  Exports the current multifile project to a xodball
+  $ xodc resave . -o ~/foo.xodball
+
+  Outputs the current multifile project as a xodball to stdout
+  $ xodc resave
+
+  Resaves one xodball into another (useful for applying migrations)
+  $ xodc resave foo.xodball -o bar.xodball
+
+  Converts a xodball to a multifile project
+  $ xodc resave foo.xodball -o /some/new/dir
+```
+
+_See code: [src/commands/resave.js](https://github.com/xodio/xod/blob/master/packages/xod-cli/src/commands/resave.js)_
+
+## `xodc tabtest [options] [entrypoint]`
+
+tabtest project
+
 ```
+USAGE
+  $ xodc tabtest [options] [entrypoint]
+
+ARGUMENTS
+  ENTRYPOINT
+      Project and/or patch to operate on. The project should point to a file or
+      directory on the file system. The patch may either point to file system or
+      be a XOD patch path. If either is omitted, it is inferred from the current
+      working directory or another argument. Examples:
+
+         * ./path/to/proj.xodball main      # xodball + patch name
+         * ./path/to/proj/main/patch.xodp   # just full path to a patch
+         * main                             # a patch in the current project
+
+OPTIONS
+  -V, --version                show CLI version
+  -h, --help                   show CLI help
+  -o, --output-dir=output-dir  [default: /tmp/xod-tabtest] path to directory where to save tabtest data
+  -q, --quiet                  do not log messages other than errors
+  -w, --workspace=workspace    [default: /home/sk/xod] use the workspace specified, defaults to $HOME/xod
+  --no-build                   do not build
+
+EXAMPLES
+  Build tabtests for project in current working directory
+  $ xodc tabtest
+  Specify target directory and project, only generate tests
+  $ xodc tabtest --no-build --output-dir=/tmp/xod-tabtest ./workspace/__lib__/xod/net
```

Comment:
You’ve split examples with an empty line for some commands and it looks very nice. Suggest applying the same here and in all examples which are glued together.